### ISSUE TITLE
how to set global tslint.json by url?

### ISSUE THREAD

**Issue by stepancar** [AtomLinter/linter-tslint#69]
Hello! Thank you for this project. 
We created public service which response our common tslint.json
In webstorm tslint we can provide http path to our tslint, but in atom we cant/ how to do?
Thank you!

**Comment by rederteph** [comment]
It would be cool to be able to set path to tslint.json from config.

**Comment by arusakov** [comment]
@stepancar Http path (url) to our ```tslint.json```, isn't it?
@rederteph Config path to ```tslint.json``` is a good feature. I'm going to start working on this tonight or tomorrow.

**Comment by stepancar** [comment]
@arusakov , yes, tslint.json. it's good idea when you wanna share one tslint.json between all team members. You can place it on any website in private net of company. Or you can place it on shared disc. If you will implement path to tslint.json feature, please, check that net paths like (\\192.168.100.*\folder\tslint.json) is available. Thank you!

**Comment by glen-84** [comment]
@arusakov Did you make any progress on this? (the `configFile` setting)

**Comment by miroswan** [comment]
I think it would make sense if the plugin searched for a tslint file within the project first, then check for a global config file.
